packages/gated-content/src/GatedClient.ts

```diff
@@ -72,6 +85,7 @@ export class GatedClient {
     this.signer = signer;
     this.storage = createAuthStorage(storageProvider, environment.name);
     this.encryptionProvider = encryptionProvider;
+    this.litClient = new NodeClient({ debug: false });
```

Comment:
wondering if we could allow it to be passed from the constructor as looking into the code on the app side 
```
 constructor(config: GatedClientConfig) {
        super(config);

        this.litClient = litClient;
      }
```
we create `NodeClient` just to replace it on the next line with out custom react-native implmentation.

What if we go with:
```
constructor({
    authentication,
    environment,
    signer,
    storageProvider,
    encryptionProvider,
    litClient = new NodeClient({debug: false}),
  }: GatedClientConfig) {
```
@cesarenaldi @paulxuca  wdyt?

Reply:
Let's try to wrap this up for now, compared to using the `IEncryptionProvider` and the cloned `GatedClient` solution this is already an order of magnitude better IMO.